index.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
+<html>
+<head>
+<title>Nicholas Botorog</title>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="stylesheet" href="style.css" >
+<script src="https://kit.fontawesome.com/00804bacca.js" crossorigin="annonymous"></script>
+</head>
+<body>
+    <nav class="navbar">
```

Comment:
Use the <header> element to wrap all other elements that make up your header ( like the <nav> or similar